assets/js/base/components/checkbox-control/stories/index.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * External dependencies
+ */
+import { text } from '@storybook/addon-knobs';
+import { useState } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Component from '../';
```

Comment:
It seems like in other stories we are importing the components with their name (in this case, it would be `CheckboxControl`), and the story has a generic name (in some that I checked, [it was named `Default`](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/trunk/assets/js/base/components/read-more/stories/index.js#L11)). Do you think we should keep it consistent here?

Reply:
Yes I agree! I'll make this change